Traceback (most recent call last):
  File "C:\Python\Python310\lib\unittest\runner.py", line 176, in run
    test(result)
  File "C:\Python\Python310\lib\unittest\suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "C:\Python\Python310\lib\unittest\suite.py", line 122, in run
    test(result)
  File "C:\Python\Python310\lib\unittest\suite.py", line 84, in __call__
    return self.run(*args, **kwds)
  File "C:\Python\Python310\lib\unittest\suite.py", line 122, in run
    test(result)
  File "C:\Python\Python310\lib\unittest\case.py", line 650, in __call__
    return self.run(*args, **kwds)
  File "C:\Python\Python310\lib\unittest\case.py", line 620, in run
    result.stopTest(self)
  File "C:\Program Files\JetBrains\PyCharm 2021.2.3\plugins\python\helpers\pycharm\teamcity\unittestpy.py", line 263, in stopTest
    output = sys.stdout.getvalue()
AttributeError: '_io.TextIOWrapper' object has no attribute 'getvalue'
